### ISSUE TITLE
can we use an pre-trained embedding instead of training from scratch?

### ISSUE THREAD

**Issue by joyousrabbit** [yunjey/show-attend-and-tell-tensorflow#11]
Hello, at the very beginning of input, it's a embedding for word2vec from scratch. Can we use an pre-trained embedding instead of training it from scratch?

**Comment by kilickaya** [comment]
Yes you can use. You can create your own vocabulary, and then extract their own word2vec representation using any trained embedding, like Google News via Gensim. 

Then, at training/testing time, you should set the trainable parameter to false for the word embedding. Then, you should load the pre-trained embedding to initialize the embedding lookup table that is created:

session.run(word2vec.assign(my_own_word2vec_matrix))

But bear in mind that it will not result in a better performance. These embeddings are trained with different objectives in mind..

**Comment by joyousrabbit** [comment]
Thank you for reply. And why an embedding for encoding, but no embedding
for decoding?
